tests/Unit/inc/Engine/Optimization/Minify/JS/Minify/optimize.php

```diff
@@ -63,5 +63,10 @@ public function testShouldMinifyJS( $original, $minified, $cdn_hosts, $cdn_url,
 			$minified,
 			$this->minify->optimize( $original )
 		);
+
+		$this->assertTrue( $this->filesystem->exists( 'wordpress/wp-content/cache/min/1/wp-content/themes/twentytwenty/assets/script-09b5ce74889313bd51265ef983880c47.js' ) );
```

Comment:
The same in here, the JS Minify files can be moved in Fixtures in an array. What do you think?